packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -1709,18 +1710,44 @@ test('mutable liquidity triggers and interest', async t => {
   for (let i = 0; i < 8; i += 1) {
     manualTimer.tick();
   }
-  await waitForPromisesToSettle();
+  t.is(bobUpdate.value.vaultState, Phase.ACTIVE);
+  trace(
+    t,
+    'bob active 2?',
+    bobUpdate.value.vaultState,
+    await E(bobVault).getCurrentDebt(),
+  );
+
   aliceUpdate = await E(aliceNotifier).getUpdateSince(aliceUpdate.updateCount);
-  bobUpdate = await E(bobNotifier).getUpdateSince(bobUpdate.updateCount);
   t.is(aliceUpdate.value.vaultState, Phase.LIQUIDATED);
+  trace(t, 'alice liquidated');
 
-  for (let i = 0; i < 5; i += 1) {
-    manualTimer.tick();
-  }
-  await waitForPromisesToSettle();
-  bobUpdate = await E(bobNotifier).getUpdateSince(bobUpdate.updateCount);
+  bobUpdate = await E(bobNotifier).getUpdateSince();
+  trace(
+    t,
+    'bob state?',
+    bobUpdate.value.vaultState,
+    await E(bobVault).getCurrentDebt(),
+  );
+  // 5 days pass
+  manualTimer.tick();
+  manualTimer.tick();
+  manualTimer.tick();
+  manualTimer.tick();
+  await manualTimer.tick();
 
+  bobUpdate = await E(bobNotifier).getUpdateSince();
+  trace(
+    t,
+    'bob 2 state?',
+    bobUpdate.value.vaultState,
+    await E(bobVault).getCurrentDebt(),
+  );
+
+  await waitForPromisesToSettle();
+  bobUpdate = await E(bobNotifier).getUpdateSince();
```

Comment:
I suspect that if you want for a fresh update you won't have to mess with settling promises.
```suggestion
  bobUpdate = await E(bobNotifier).getUpdateSince(bobUpdate.updateCount);
```